src/atoms/Checkbox/Checkbox.jsx

```diff
@@ -19,25 +19,28 @@ const Checkbox = ({
 	</div>
 );
 
-Checkbox.defaultProps = {};
+Checkbox.defaultProps = {
+	callback: () => {},
+	index: 1,
+};
 
 Checkbox.propTypes = {
 	callback: PropTypes.func,
-	children: PropTypes.node,
 	index: PropTypes.number,
 	selected: PropTypes.bool,
 	value: PropTypes.oneOfType([
 		PropTypes.string,
 		PropTypes.number,
 	]).isRequired,
+	label: PropTypes.string,
 };
 
 export default Checkbox;
 
 /*
 options:
 	callback: optional callback on selection
-	children: required for displaying a label
+	label: required for displaying a label
```

Comment:
Worth mentioning that having child nodes as props makes the component much more customizable, as anything that can be rendered will be an accepted PropType. Probably not something we need here though, so having an enforced label might be the right move :)